scripts/Auth_scripts/auth.ps1

```diff
@@ -0,0 +1,36 @@
+Function authUser($url)

```

Comment:
Lets declare the exact type for the $url variable - function AuthUser([string]$urlDomain)
Please, rename your public functions to start with capital letter.

Reply:
Changed.